algorithms/indexing/lattice_search/__init__.py

```diff
@@ -83,15 +83,15 @@
     pkg_resources.iter_entry_points("dials.index.basis_vector_search"),
     pkg_resources.iter_entry_points("dials.index.lattice_search"),
 ):
-    scope = (
-        """\
-%s {
-    include scope entry_point.load().phil_scope
-}
-    """
-        % entry_point.name
+    ext_master_scope = libtbx.phil.parse(
+        """
+%s
+.expert_level=1
+.help=%s
+{}
+        """
+        % (entry_point.name, entry_point.load().phil_help)
```

Comment:
Are we content that, since `dials.algorithms.indexing.basis_vector_search.strategy.Strategy` and `dials.algorithms.indexing.lattice_search.strategy.Strategy` both now have the attribute `phil_help = "''"`, we can safely assume that `entry_point.load().phil_help` will always exist?  Is anyone ever going to create an indexing strategy class that doesn't subclass one of these `Strategy` base classes?  If the answer to the latter question is yes, then we probably want to include the following too.
```suggestion
    try:
        phil_help = entry_point.load().phil_help
    except AttributeError:
        phil_help = "''"

    ext_master_scope = libtbx.phil.parse(
        """
%s
.expert_level=1
.help=%s
{}
        """
        % (entry_point.name, entry_point.load().phil_help)
```

Reply:
I think it is safe to enforce that any new strategies should inherit from the existing `Strategy` base classes.